bin/xpm.dart

```diff
@@ -25,7 +26,16 @@ void main(List<String> args) async {
     showVersion(args);
   }
 
+  final refresh = 'automatic_refresh';
+  final bool refreshExpired = await Setting.get(refresh, defaultValue: false);
```

Comment:
O nome dessa variável tá invertido ou errado. ! significa não, e você fez um if not refresh expired, que não é verdade pra questão. Poderia ser algo:
final bool isExpired = await Setting.get(refresh, defaultValue: true);

Nota também que mudei o valor padrão pra `true`. Afinal, se essa variável não existe, é porque tá faltando refresh.